core/ui/compose/designsystem/src/test/kotlin/app/k9mail/core/ui/compose/designsystem/atom/textfield/TextInputTextFieldTest.kt

```diff
@@ -60,6 +84,7 @@ class TextInputTextFieldTest(
                     TextFieldOutlined(
                         value = value,
                         onValueChange = onValueChange,
+                        isSingleLine = true,
```

Comment:
I think there should be an additional test case, one with single line enabled and one without.